cmd/geth/consolecmd.go

```diff
@@ -231,13 +229,10 @@ func ephemeralConsole(ctx *cli.Context) error {
 			utils.Fatalf("Failed to execute %s: %v", file, err)
 		}
 	}
-	// Wait for pending callbacks, but stop for Ctrl-C.
-	abort := make(chan os.Signal, 1)
-	signal.Notify(abort, syscall.SIGINT, syscall.SIGTERM)
 
 	go func() {
-		<-abort
-		os.Exit(0)
```

Comment:
Was the issue indeed the `os.Exit(0)` on latest go?

Reply:
No... it was the `snappy` library. https://github.com/etclabscore/core-geth/commit/69c9cc008d4b742dddb31f138b201547e91812ab